aptos-move/framework/src/aptos.rs

```diff
@@ -71,6 +71,7 @@ impl ReleaseTarget {
                 "aptos-token-objects",
                 Some("cached-packages/src/aptos_token_objects_sdk_builder.rs"),
             ),
+            ("aptos-experimental", None),
```

Comment:
We are sure this won't get shipped to `testnet`/`mainnet` here?

Reply:
we can run a command to generate  release artifacts for testnet, to confirm.